src/apis/eduidPersonalData.ts

```diff
@@ -24,6 +24,12 @@ export interface UserNameRequest {
   given_name: string;
   surname: string;
   chosen_given_name?: string;
+}
+
+export interface UserNameResponse {
```

Comment:
maybe we can use extends here to avoid duplicating code? 
```
export interface UserNameResponse extends UserNameRequest {
  legal_name?: string;
}
```